content/en/docs/reference/config/config-status/index.md

```diff
@@ -0,0 +1,104 @@
+weight: 21
+---
+
+{{< warning >}}
+This feature is in the Alpha stage, see
+[Istio Feature Status](/about/feature-stages/). Your feedback is welcome in the
+[Istio User Experience discussion](https://discuss.istio.io/c/UX/23). Currently,
+this feature is tested only for single, low volume clusters with a single
+control plane revision.
+{{< /warning >}}
+
+Istio 1.6 and later provides information about the propagation of configuration
+changes through the mesh, using the `status` field in a resource's
+[`CustomResourceDefinition`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+(CRD).
+
+The `status` field contains the state of a resource's configuration with various
+informational messages, including:
+
+* The resource's readiness.
+* How many data plane instances are associated with it.
+* Information for the output of tools, such as `istioctl analyze`.
+
+For example, the `kubectl wait` command monitors the `status` field to determine
+whether to unblock configuration and resume. For more information, see
+[Wait for Resource Status to Apply Configuration](/docs/ops/configuration/mesh/config-resource-ready/).
+
+## View the `status` field
+
+To view the contents of the `status` field, use the following command:
+
+{{< text bash >}}
+$ kubectl get virtualservice service-name
+{{< /text >}}
+
+In the output, the `status` field contains several child fields with details
+about the process of propagating configuration changes through the mesh. A
+child field for the condition status, also named `status`, has a value of `True`
+when Istio applied a configuration successfully. A value of `False` indicates
+there was an error.
+
+{{< text yaml >}}
+status:
+  conditions:
+  - lastProbeTime: null
+    lastTransitionTime: "2019-12-26T22:06:34Z"
+    message: "61/122 complete"
+    reason: "stillPropagating"
+    status: "False"
+    type: Reconciled
+  - lastProbeTime: null
+    lastTransitionTime: "2019-12-26T22:06:56Z"
+    message: "1 Error and 1 Warning found. See validationMessages field for details"
+    reason: "errorsFound"
+    status: "False"
+    type: PassedAnalysis
+  validationMessages:
+  - code: IST0101
+    level: Error
+    message: 'Referenced gateway not found: "bogus-gateway"'
+  - code: IST0102
+    level: Warn
+    message: 'mTLS not enabled for virtual service'
+{{< /text >}}
+
+## The `type` and `status` field workflow
+
+The `type` field indicates the configuration condition. It can have the
+following values:
+
+* `PassedAnalysis`
+* `Reconciled`
+
+When you apply a configuration, the `type` field immediately sets to
+`Reconciled` and `status` to `False`, which blocks further commands until
+`status` equals `True` or the time out period expires. The `status` field might
+transition beyond `False` instantaneously, depending on the speed of the
+cluster.
+
+In parallel, the `PassedAnalysis` field is set to `True` or `False` to indicate
```

Comment:
PassedAnalysis is not a field, it's a value.

Reply:
"The condition of type 'PassedAnalysis' has a status of 'True' or 'False' to..."

Reply:
fixed